tools/ci-scripts/linux/image-builder/README.md

```diff
@@ -1,52 +1,75 @@
+This directory contains the source for a Docker container that is used for
+building Bond.
 
-# Pre-requisites
+The image is based on Ubuntu 16.04 (Xenial Xerus) and contains:
 
-Install the following pre-requisites:
+* pre-installed build tools for the various languages Bond supports,
+* binaries of a number of Boost versions for the C++ and Python builds, and
+* an entrypoint that can be used to drive a build.
 
-* [Azure CLI tools][azure-cli]
-* docker
-* zsh
+# Automatic Builds
 
-For deb-based distributions, docker and zsh can be installed via:
+![Docker Ubuntu image build status](https://msazure.visualstudio.com/_apis/public/build/definitions/b32aa71e-8ed2-41b2-9d77-5bc261222004/14573/badge)
 
-    $ sudo apt update && sudo apt install docker.io zsh
+A new image is automatically built when the master branch of the main Bond
+repository (https://github.com/Microsoft/bond) is changed. These images are
+then pushed to the bondciimages.azurecr.io/ubuntu-1604 container repository.
+The Travis CI builds pull fixed versions of the image and use that as their
+basis.
+
+The image is built on a Microsoft VSTS instance. It's status is shown in the
+image above. Currently, access to the results of this build are limited to
+Microsoft employees. If you need details about a given build, please
+[open an issue](https://github.com/Microsoft/bond/issues/new) and one of the
+maintianers from Microsoft can get you the details.
+
+# Manual Builds
 
-# Building the Image
+You can build and use the image locally.
 
-To create the image, start in the directory containing this README.
+## Pre-requisites
 
-Run the `build_image.zsh` script which will make the docker image with all
-required dependencies and will compile all required versions of Boost (using
-the `build_boosts.zsh` script) that will become part of the image:
+Install the following pre-requisites:
+
+* Docker
+* zsh
 
-    $ ./build_image.zsh
+For deb-based distributions, Docker and zsh can be installed via:
+
+    $ sudo apt update && sudo apt install docker.io zsh
+    
+zsh is only needed to debug and work on `build_boost.zsh` locally.
 
-The build takes about 50 minutes on a quad-core 2.4 GHz Xeon. When it finishes,
-there will be a `bond-xenial.tar.gz` archive in this directory.
+## Building the Image
 
-# Uploading the Image
+To create the image, start in the directory containing this README and run:
 
-After the image archive has been created, it needs to be uploaded to the
-Azure storage account that the Travis CI builds use.
+    $ docker build -t bond-ubuntu-1604 .
 
-## Credentials
+This will make the Docker image with all the required dependencies and will
+compile all the required versions of Boost (using the `build_boosts.zsh`
+script) that will become part of the image.
 
-Ensure the file `~/.azure/bondbinaries.sh` has the current Azure storage
-account details:
+The build takes about 50 minutes on a quad-core 2.4 GHz Xeon. When it
```

Comment:
This sounds a lot worse than it is. I'll time a run on a different machine for this.

Reply:
18:42 on a quad-core 3.3 GHz Skylake Xeon. (Extra specifically, E3-1225 v5.)